### PR TITLE
TILA-1357 | Application api

### PR DESCRIPTION
**Pull request by Niilis**
Expose Unit through Application API.

Refs TILA-1357

### CODE REVIEW

File: api/reservation_units_api.py [hunk 1]
```diff
@@ -146,6 +146,11 @@ class ReservationUnitSerializer(TranslatedModelSerializer):
     unit_id = serializers.PrimaryKeyRelatedField(
         queryset=Unit.objects.all(), source="unit"
     )
+    unit_details = UnitSerializer(
```
Comment (nicobav) [inline api/reservation_units_api.py:149]: Actually we already have unit here in `ReservationUnitSerializer`- it was added just recently

Comment (nicobav) [reply api/reservation_units_api.py:149]: [here](https://github.com/City-of-Helsinki/tilavarauspalvelu-core/pull/462/files#diff-a6d1d83bc329f54c8f8a659daf844738e39aeb173daf244a794327e2578d608dR157)

Comment (Niilis) [reply api/reservation_units_api.py:149]: Ah, we seem to have had duplicate tickets created for this! I'll close this MR.

**Comment by Niilis** [comment]
Duplicate ticket TILA-1363 already completed.

**Pull request closed without merging**